### ISSUE TITLE
Bug

### ISSUE THREAD

**Issue by Torbeckuz** [THEREALWWEFAN231/Inertia#867]
**Inertia Utility Mod version:** 3.1.3
**Minecraft version:** 1.16.4
**Operating system:** 
**Optifine/Other mods:** N/A (Running Fabritone=

**What's the issue:** 
Going Freecam and the pressing F1 for the cinematic view makes the screen pitch black
**Crash-report(Would be very useful also please format):**

**Comment by THEREALWWEFAN231** [comment]
Do you have Name Tags and or Combat Info on?

**Comment by Torbeckuz** [comment]
Yes, I had nametags with Hearts Head and Items with enchantments. I didn't have combat info

**Comment by feliscatusmeows** [comment]
try disabling nametags
